src/Framework/NativeMethods.cs

```diff
@@ -959,7 +959,7 @@ internal static string GetShortFilePath(string path)
 
             if (length > 0)
             {
-                StringBuilder fullPathBuffer = new StringBuilder(length);
+                char[] fullPathBuffer = new char[length];
```

Comment:
The `ToString` call on line 968 will return `"System.Char[]"`. You'll have to use `new string(...)` instead.